gnd/src/main/java/com/google/android/gnd/ui/common/BindingAdapters.java

```diff
@@ -118,4 +129,13 @@ public static void setOnFocusChangeListener(
   public static void setOnShowDialogListener(MultipleChoiceFieldLayout view, Runnable listener) {
     view.setOnShowDialogListener(listener);
   }
+
+  @BindingAdapter("bind:uri")
+  public static void bindUri(ImageView view, Uri uri) {
+    if (uri == null) {
+      return;
```

Comment:
Ideally, when this happens the visibility of MaterialCard should be set to GONE

Reply:
When could this actually be null? If it happens, the widget's previous state will remain. At some point, when it's made visible, the old state will show even though Uri is null, which is obviously incorrect. It would seem we'd want the state of the widget to match to current field binding, but I may be missing a piece!